FND/Places/NorthAmerica/USPostalServiceAddressesIndividuals.rdf

```diff
@@ -0,0 +1,2119 @@
+		<fibo-fnd-utl-av:commonDesignation>CROSSROADS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>XRDS</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Curve">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>curve</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>CURVE</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>CURV</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Dale">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>dale</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>DALE</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DL</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>DL</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Dam">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>dam</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>DAM</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DM</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>DM</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Divide">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>divide</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>DIV</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DIVIDE</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DV</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DVD</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>DV</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Drive">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>drive</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>DR</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DRIV</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DRIVE</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DRV</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>DR</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Drives">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>drives</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>DRIVES</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>DRS</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Estate">
```

Comment:
This is likely to clash with what people leave to their survivors when they die

Reply:
We have said that we did not need unique names for individuals as long as the namespace was clear.